packages/starlight/schemas/social.ts

```diff
@@ -97,6 +98,7 @@ export const SocialLinksSchema = () =>
 					confluence: 'Confluence',
 					jira: 'Jira',
 					storybook: 'Storybook',
+					npm: 'npm',
```

Comment:
I don’t know for sure because in testing with VoiceOver on macOS both “npm” and “NPM” were both read correctly, but I wonder if we should use the uppercase acronym style for the accessible label in case there are any screen readers that take that as a hint to read the letters separately vs trying to pronounce it as a word?

```suggestion
					npm: 'NPM',
```

Reply:
Personally I cringe a bit at seeing uppercased npm (also see their about page), but I'm not an a11y expert so if you insist I'm happy to update it.

Reply:
Yeah. To be clear this text is never displayed visibly in UI, it is only used to provide an accessible label for those links. @HiDeoo was kind enough to test in NVDA and JAWS which also pronounce it correctly with the lowercase version, so I think we’re fine to leave it as-is and avoid you having to cringe 😄

Reply:
Sorry, that was an unfortunate way to express just personally not loving to see npm spelled uppercased, like, anywhere. But this isn't about me, it's about accessibility. Glad it's pronounced properly by assistive tech! Thanks to both of you for some high quality QA here ❤️